api/v1/atlasintegration_types.go

```diff
@@ -0,0 +1,187 @@
+/*
+Copyright 2025 MongoDB.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/api/v1/status"
+)
+
+func init() {
+	SchemeBuilder.Register(&AtlasIntegration{}, &AtlasIntegrationList{})
+}
+
+// +kubebuilder:object:root=true
+
+// AtlasIntegration is the Schema for the atlas 3rd party inegrations API.
+type AtlasIntegration struct {
```

Comment:
The resource in the API is called Third-Party Integration, I'd rename the resource to `AtlasThirdPartyIntegration`

Reply:
👍

Reply:
generated file names normally match the types they define. To keep the standard, please rename any files named `atlasintegration` to `atlasthirdpartyintegration`.